.travis.yml

```diff
@@ -6,14 +6,6 @@ cache:
   directories:
     - $HOME/.composer/cache/files
 
-php:
-  - 5.3
-  - 5.4
-  - 5.5
-  - 7.0
```

Comment:
this means not having any job testing with any of the above listed PHP versions

Reply:
Hi @xabbuh, I'm not an expert in Travis, but I noticed that previously they was tested twice (see https://travis-ci.org/michelsalib/BCCCronManagerBundle/builds/304995418), since now they are all tested only once, following the matrix (https://travis-ci.org/michelsalib/BCCCronManagerBundle/builds/304997530).

Reply:
Actually you're right, my mistake, the versions are not doubled, so I put them back. Thanks ;)